TestAssets/TestProjects/SimpleDependencies/SimpleDependencies.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.0-beta-001507-00</RuntimeFrameworkVersion>
```

Comment:
@eerhardt is there a good way to specify this to pickup 2.0.0 or higher ?
By default i see that tests are built against 1.0